src/tracer.cc

```diff
@@ -477,6 +460,45 @@ TraceManager::CleanupTracer()
 }
 
 #ifndef _WIN32
+void
+TraceManager::ProcessOpenTelemetryParameters(
+    const triton::server::TraceConfigMap& config_map,
+    otlp::OtlpHttpExporterOptions& exporter_options,
+    otel_resource::ResourceAttributes& attributes,
+    otel_trace_sdk::BatchSpanProcessorOptions& processor_options)
+{
+  attributes[otel_resource::SemanticConventions::kServiceName] =
+      std::string("triton-inference-server");
+  auto mode_key = std::to_string(TRACE_MODE_OPENTELEMETRY);
+  auto otel_options_it = config_map.find(mode_key);
+  if (otel_options_it != config_map.end()) {
```

Comment:
nit: It can help reduce nesting to check the inverse and exit early:
```cpp
if (otel_options_it == config_map.end()) {
  return;
}

for (...) {
}
```